cloud/aws/bin/deploy

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+set -e
+
+source "cloud/shared/bin/lib.sh"
+source "cloud/azure/bin/lib.sh"
```

Comment:
Should this be importing anything from azure?

Reply:
good catch. we don't need this.